src/components/common/CourseOverview/courseOverview.tsx

```diff
@@ -242,7 +242,6 @@ const CourseOverview = ({ course, grades }: CourseOverviewProps) => {
           <Skeleton />
         </p>
         <p className="text-lg font-semibold text-center">
-          {searchQueryLabel(course)}
```

Comment:
you'll want to replace this with a `<Skeleton />` instead of just removing it so that it shows up as a loader

once you do that you may notice it spans the whole width instead of being about the width of say the text "GOVT 2306" so you'll need to limit it in a similar way to some of the below skeletons